int8 paramid
int32 intValue
float64 doubleValue
string strValue
bool boolValue
int8 ftprintno
float64[] footprintlist
